apps/cowswap-frontend/src/modules/application/containers/App/menuConsts.tsx

```diff
@@ -41,9 +41,7 @@ export const NAV_ITEMS: MenuItem[] = [
   },
   {
     label: 'More',
-    badge: 'New',
-    badgeType: BadgeTypes.ALERT,
-    children: [
+   children: [
```

Comment:
Missing an indentation space

```suggestion
    children: [
```